test/integration/master/kms_transformation_test.go

```diff
@@ -257,7 +257,6 @@ resources:
 
 func mustBeHealthy(t *testing.T, checkName string, clientConfig *rest.Config) {
 	t.Helper()
-	var restErr error
```

Comment:
I think this was trying to capture the inner error to give more useful debugging messages if the poll timed out. Suggest assigning the inner err to restErr

Reply:
Thank you,I adjusted the wrong order of printing and clear description added.
Is it ok?